src/api/spec/models/bs_request_spec.rb

```diff
@@ -5,6 +5,7 @@
 # require real backend answers for projects/packages.
 # CONFIG['global_write_through'] = true
 
+# rubocop:disable Metrics/BlockLength
```

Comment:
what about following the advice instead of disabling it or disable it generally as this may happen in rspec generally? :thinking:

Reply:
Disabling it for rspec generally sounds good - but outside of this PR.